### PR TITLE
docs(meeting): added 04-22meeting.md

### PR DESCRIPTION
**Pull request by hhundhausen**

### CODE REVIEW

File: admin/meetings/0420-meeting.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: <04-20-23> Popup Thursday Meeting
```
Comment (Kiyo5hi) [inline admin/meetings/0420-meeting.md:2]: 2023-04-20
this format is required, refer to `.meeting-template.md`

File: admin/meetings/0420-meeting.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+type: <routine | popup | others>
```
Comment (Kiyo5hi) [inline admin/meetings/0420-meeting.md]: `type: routine`

angle brackets mean required arguments, and lines mean OR

File: admin/meetings/0420-meeting.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+date: <04-20-23>
```
Comment (Kiyo5hi) [inline admin/meetings/0420-meeting.md]: 2023-04-20
this format is required, refer to `.meeting-template.md`

File: admin/meetings/0420-meeting.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+time: <6:20> - <8:10>
```
Comment (Kiyo5hi) [inline admin/meetings/0420-meeting.md]: remove angle brackets

File: admin/meetings/0420-meeting.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+method: <in-person | virtual | hybrid>
```
Comment (Kiyo5hi) [inline admin/meetings/0420-meeting.md]: `method: in-person`

File: admin/meetings/0422-meeting.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
+---
+title: <04-22-23>  Saturday Meeting 
+type: <routine> 
+date: <4-22-23>
+time: <1:00PM> - <time>
+method: <virtual>
+---
```
Comment (Kiyo5hi) [inline admin/meetings/0422-meeting.md:7]: +1 refer prev. comments

File: admin/meetings/meeting-template.md [hunk 7]
```diff
@@ -6,46 +6,74 @@ time: <time> - <time>
 method: <in-person | virtual | hybrid>
 ---
 
+#Meeting Title
```
Comment (Kiyo5hi) [inline admin/meetings/meeting-template.md]: nit: add a space after the hash tag

**Review by Kiyo5hi — COMMENTED** [review]
Thanks for composing all these!

**Comment by hhundhausen** [comment]
Updated branch with changes and added assignment files

**Pull request closed without merging**